scripts/pysa_fuzzer/run2.py

```diff
@@ -0,0 +1,187 @@
+import subprocess
```

Comment:
could we unify run.py and run2.py? Many of the functions are similar. We just need to have different functions for the code generation. There could be a command line parameter to pick the code generator.